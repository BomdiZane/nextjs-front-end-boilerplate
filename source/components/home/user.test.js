import Users from './users';

describe('Users component:', () => {
	const users = shallow(<Users store={ store } />);

	it('should match snapshot', () => expect(users).toMatchSnapshot());
});
