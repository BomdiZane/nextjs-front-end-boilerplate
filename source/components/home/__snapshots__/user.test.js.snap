// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Users component: should match snapshot 1`] = `
<Query
  displayName="Apollo(UsersHolder)"
  query={
    Object {
      "definitions": Array [
        Object {
          "directives": Array [],
          "kind": "OperationDefinition",
          "name": undefined,
          "operation": "query",
          "selectionSet": Object {
            "kind": "SelectionSet",
            "selections": Array [
              Object {
                "alias": undefined,
                "arguments": Array [],
                "directives": Array [],
                "kind": "Field",
                "name": Object {
                  "kind": "Name",
                  "value": "users",
                },
                "selectionSet": Object {
                  "kind": "SelectionSet",
                  "selections": Array [
                    Object {
                      "alias": undefined,
                      "arguments": Array [],
                      "directives": Array [],
                      "kind": "Field",
                      "name": Object {
                        "kind": "Name",
                        "value": "id",
                      },
                      "selectionSet": undefined,
                    },
                    Object {
                      "alias": undefined,
                      "arguments": Array [],
                      "directives": Array [],
                      "kind": "Field",
                      "name": Object {
                        "kind": "Name",
                        "value": "firstName",
                      },
                      "selectionSet": undefined,
                    },
                    Object {
                      "alias": undefined,
                      "arguments": Array [],
                      "directives": Array [],
                      "kind": "Field",
                      "name": Object {
                        "kind": "Name",
                        "value": "lastName",
                      },
                      "selectionSet": undefined,
                    },
                    Object {
                      "alias": undefined,
                      "arguments": Array [],
                      "directives": Array [],
                      "kind": "Field",
                      "name": Object {
                        "kind": "Name",
                        "value": "bags",
                      },
                      "selectionSet": undefined,
                    },
                  ],
                },
              },
            ],
          },
          "variableDefinitions": Array [],
        },
      ],
      "kind": "Document",
      "loc": Object {
        "end": 59,
        "start": 0,
      },
    }
  }
  skip={false}
  warnUnhandledError={true}
>
  <Component />
</Query>
`;
